apps/api-extractor/src/analyzer/AstSymbolTable.ts

```diff
@@ -210,6 +211,76 @@ export class AstSymbolTable {
     return this._entitiesByIdentifierNode.get(identifier);
   }
 
+  /**
+   * Builds an AstSymbol.localName for a given ts.Symbol.  In the current implementation, the localName is
+   * a TypeScript-like expression that may be a string literal or ECMAScript symbol expression.
+   *
+   * ```ts
+   * class X {
+   *   // localName="identifier"
+   *   public identifier: number = 1;
+   *   // localName="\"identifier\""
+   *   public "quoted string!": number = 2;
+   *   // localName="[MyNamespace.MySymbol]"
+   *   public [MyNamespace.MySymbol]: number = 3;
+   * }
+   * ```
+   */
+  public static getLocalNameForSymbol(symbol: ts.Symbol): string {
+    const symbolName: string = symbol.name;
+
+    if (TypeScriptHelpers.isWellKnownSymbolName(symbolName)) {
+      // TypeScript binds well-known ECMAScript symbols like "Symbol.iterator" as "__@iterator".
+      // This converts a string like "__@iterator" into the property name "[Symbol.iterator]".
+      return `[Symbol.${symbolName.slice(3)}]`;
```

Comment:
Do these always start with `__@`? Is that where the `3` comes from? Can you be a little more explicit about that in the docs?

Reply:
Good point. I moved this code into a proper API and documented it better.